src/backend/joanie/core/templatetags/extra_tags.py

```diff
@@ -36,3 +36,22 @@ def list_key(items: list[dict], key: str):
     A template tag filter to get a list of values from a list of dictionaries.
     """
     return [item[key] for item in items]
+
+
+@register.filter
+def iso8601_to(duration, unit):
```

Comment:
to what ? You function name is not relevant. you can call it `iso8601_to_duration` and rename your import to avoid conflict